pkg/cafs/cafs.go

```diff
@@ -148,8 +148,8 @@ func (d *defaultFs) writer(prefix string) Writer {
 		pather:        nil,
 		prefix:        prefix,
 		count:         0,
-		flushChan:     make(chan blobFlush, 100000),
-		errC:          make(chan error, 1000000),
+		flushChan:     make(chan blobFlush, sizeOfFlushChan),
+		errC:          make(chan error, 1),
```

Comment:
in case of more than two errors, this'll leave goroutines parked on write to the error channel.

Reply:
Yes we need to fix #171